examples/calling-apis/chatbot/app/(langgraph)/api/langgraph/[..._path]/route.ts

```diff
@@ -15,14 +15,33 @@ async function getAccessToken() {
 
 export const { GET, POST, PUT, PATCH, DELETE, OPTIONS, runtime } =
   initApiPassthrough({
+    disableWarningLog: true,
     apiUrl: process.env.LANGGRAPH_API_URL,
     apiKey: process.env.LANGSMITH_API_KEY,
     runtime: "edge",
     baseRoute: "langgraph/",
     headers: async (req: NextRequest) => {
```

Comment:
sorry, only now checking how the library works under the hood... you don't actually want to copy all the headers, because the library will copy them by default as we can see here: https://github.com/bracesproul/langgraph-nextjs-api-passthrough/blob/c85c0edd5544417dcfd035993718bba587e56d8e/src/index.ts#L52-L59

So `headers` is only meant to be used to _add_ headers if you need. Thus the right implementation would be:
```
headers: async (req: NextRequest) => {
  const accessToken = await getAccessToken();
  return {
    Authorization: `Bearer ${accessToken}`,
  };
},
```

this is explained in the README: https://github.com/bracesproul/langgraph-nextjs-api-passthrough/blob/c85c0edd5544417dcfd035993718bba587e56d8e/README.md#custom-headers